queueUI/resources/mainwindow.ui

```diff
@@ -53,6 +53,227 @@
+         <widget class="QFrame" name="frame">
+          <property name="geometry">
+           <rect>
+            <x>10</x>
+            <y>10</y>
+            <width>1151</width>
+            <height>181</height>
+           </rect>
+          </property>
+          <property name="frameShape">
+           <enum>QFrame::WinPanel</enum>
+          </property>
+          <property name="frameShadow">
+           <enum>QFrame::Raised</enum>
+          </property>
+          <property name="lineWidth">
+           <number>3</number>
+          </property>
+          <property name="midLineWidth">
+           <number>5</number>
+          </property>
+          <widget class="QFrame" name="hostServerConnection">
+           <property name="geometry">
+            <rect>
+             <x>20</x>
+             <y>10</y>
+             <width>201</width>
+             <height>151</height>
+            </rect>
+           </property>
+           <widget class="QPushButton" name="connectButton">
+            <property name="geometry">
+             <rect>
+              <x>10</x>
+              <y>30</y>
+              <width>181</width>
+              <height>36</height>
+             </rect>
+            </property>
+            <property name="text">
+             <string>Connect</string>
+            </property>
+           </widget>
+           <widget class="QPushButton" name="disconnectButton">
+            <property name="geometry">
+             <rect>
+              <x>10</x>
+              <y>70</y>
+              <width>181</width>
+              <height>36</height>
+             </rect>
+            </property>
+            <property name="text">
+             <string>Disconnect</string>
+            </property>
+           </widget>
+           <widget class="QLabel" name="hostServeConnectionLabel">
+            <property name="geometry">
+             <rect>
+              <x>10</x>
+              <y>10</y>
+              <width>171</width>
+              <height>19</height>
+             </rect>
+            </property>
+            <property name="text">
+             <string>Host Server Connection</string>
+            </property>
+           </widget>
+          </widget>
+          <widget class="QFrame" name="environmentControl">
+           <property name="geometry">
+            <rect>
+             <x>250</x>
+             <y>10</y>
+             <width>201</width>
+             <height>161</height>
+            </rect>
+           </property>
+           <widget class="QPushButton" name="runEngineOpenButton_3">
+            <property name="geometry">
+             <rect>
+              <x>10</x>
+              <y>30</y>
+              <width>181</width>
+              <height>36</height>
+             </rect>
+            </property>
+            <property name="text">
+             <string>Open</string>
+            </property>
+           </widget>
+           <widget class="QPushButton" name="runEngineCloseButton_3">
```

Comment:
rename object:
- avoid `_3`
- runEngine => RE (those names are already long, you can use the fact that RE is a very well known acronym)

Reply:
Change made.